src/js/components/wizard/v2/WizardStepsV2.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import 'components/wizard/WizardSteps.scss';
+
+const WizardStepsV2 = ({ steps, currentStepKey }) => {
```

Comment:
What has changed here compared to `WizardSteps`?

Reply:
I answered that question in one of the comments below: https://github.com/openboxes/openboxes/pull/4642/files#r1617300455